CONTRIBUTING.adoc

```diff
@@ -50,8 +50,9 @@ can be built locally or in the `openshift/origin-release` container environment.
 http://golang.org/[Go] programming language is only necessary for building on
 the local host.
 
-Currently, OpenShift is built with `go` 1.8 and uses Docker 1.12. The exact
-requirement for Docker is documented
+Currently, OpenShift is built with `go` 1.9 and uses Docker 1.12 on Fedora/CentOS/RHEL.
+MacOS supports builds with `go` 1.9 and Docker CE 17.09.1 - *newer versions of go and docker are not supported*.
```

Comment:
s/*newer versions of go and docker are not supported*/newer versions of go and docker are not supported